frontend/public/index.html

```diff
@@ -184,11 +184,119 @@ <h5 >Preset 1</h5>
 	
 	</div>
 	
+	<!-- Additional information about the position of the cube. -->
+		<div id="CubePositionX" style="position:fixed; left:90%; bottom:4%; color:grey;"></div>
+		<div id="CubePositionY" style="position:fixed; left:90%; bottom:8%; color:grey;"> </div>
+		
+		<!--Scripts to be used for the joystick part-->
+		<script src="http://threejs.org/build/three.min.js"></script>
```

Comment:
I think we should look for these files as a bower dependency. The system is going to be used in a mostly offline environment, so the app cannot download these files in the fly. They need to be fetched before using them.